lib/web_ui/pubspec.yaml

```diff
@@ -2,25 +2,25 @@ name: ui
 publish_to: none
 
 environment:
-  sdk: ">=2.2.2 <3.0.0"
+  sdk: ">=2.9.0-0 <3.0.0"
 
 dependencies:
-  meta: 1.1.8
+  meta: ^1.1.8
```

Comment:
do you still need meta?

Reply:
Yeah, we have tests and dev tools using it.